packages/ui/src/helpers/authenticator/facade.ts

```diff
@@ -100,11 +101,24 @@ export const getServiceContextFacade = (state: AuthMachineState) => {
     }
   })();
 
+  const authStatus = ((route) => {
+    switch (true) {
+      case route === 'idle':
+      case route === 'setup':
+        return 'pending';
```

Comment:
What does `pending` mean in this context? Should we just be looking at the existing `isPending` value to infer the value of `pending`?

Reply:
the `isPending` is used when an API call is in progress. The `pending` in this context refers to when the xState machine is still being setup and is in a `idle` or `setup` state.

Reply:
Ok I'll change to `configuring`

Reply:
d0ae2965275f7fc57e07aff02014b04d194e055c